libs/community/tests/integration_tests/vectorstores/test_bigquery_vector_search.py

```diff
@@ -0,0 +1,88 @@
+"""Test BigQuery Vector Search.
+In order to run this test, you need to install Google Cloud BigQuery SDK
+pip install google-cloud-bigquery
+Your end-user credentials would be used to make the calls (make sure you've run
+`gcloud auth login` first).
+"""
+
+import uuid
+
+import pytest
+from google.cloud import bigquery
+
+from langchain_community.vectorstores.bigquery_vector_search import BigQueryVectorSearch
+
+
+@pytest.fixture(scope="class")
+def store(request: pytest.FixtureRequest) -> BigQueryVectorSearch:
+    """BigQueryVectorStore tests context."""
+    TestBigQueryVectorStore.store = BigQueryVectorSearch(
+        dataset_name=TestBigQueryVectorStore.dataset_name
+    )
+    TestBigQueryVectorStore.store.add_texts(
+        TestBigQueryVectorStore.texts, TestBigQueryVectorStore.metadatas
+    )
+
+    def teardown() -> None:
+        bigquery.Client(location="US").delete_dataset(
+            TestBigQueryVectorStore.dataset_name,
+            delete_contents=True,
+            not_found_ok=True,
+        )
+
+    request.addfinalizer(teardown)
+    return TestBigQueryVectorStore.store
+
+
+class TestBigQueryVectorStore:
```

Comment:
I'd suggest it might be a good idea to follow the pytest semantics that is common across Langchain.

For example: https://github.com/langchain-ai/langchain/blob/master/libs/community/tests/integration_tests/llms/test_vertexai.py

Reply:
Isn't it following pytest semantics already? These tests require setup/teardown.